package.json

```diff
@@ -6,8 +6,10 @@
   "main": "app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "dev": "NODE_ENV=development nodemon app.js",
-    "start": "NODE_ENV=production node app.js"
+    "start": "NODE_ENV=production node app.js",
+    "server": "nodemon app.js",
+    "client": "npm run start --prefix client",
+    "dev": "NODE_ENV=development concurrently \"npm run server\" \"npm run client\" "
```

Comment:
hmmm doesn't it work to add `&&` between the 2 commads so they could work after the other? i'm not sure you need a package.
unless it didn't work for you?

Reply:
It did not work for me. And the package is a better way.

Reply:
@hoslack well it's not a better way cause you are installing a package, `npm run server && npm run client` also this has worked for me

Reply:
I don't have a problem with installing this package as it is popularly used. But if you want to run scripts in parallel in a UNIX environment, you could just use a single ampersand (&) and that should work

Reply:
Okay